suite/tests/linux/clone.c

```diff
@@ -351,10 +351,13 @@ delete_thread(pid_t pid, void *stack)
 {
```

Comment:
> If the child crashes or exits with a non-zero exit code the test would otherwise get marked as passing.

More context would help: in particular, which test is this.  Someone looking at the commit history will have no idea what "the test" is, or in the title what part of the code this is referring to.  I would put the test name "linux.clone" in both the title (maybe "Text exit status of child in linux.clone test" as the title) and the description ("In the linux.clone test, ...").